modules/security-context-constraints-psa-coexistence.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="security-context-constraints-psa-coexistence_{context}"]
+= Pod security admission and security context constraints
+
+The Kubernetes pod security admission controller enforces pod security profiles. The security context constraints controller enforces security context constraints (SCC). The controllers work together using the following webhooks to enforce security policies and preserve backward compatibility:
+
+. The security context constraint mutating webhook sets the required defaults for the pod's assigned SCC. For example, if the seccomp profile is empty or not set, the webhook sets the default type to `RuntimeDefault`.
```

Comment:
This should be:
The security context constraints admission is capable of mutating some security context fields depending on which SCC matches the security contexts of a pod.

Reply:
I've tried to merge the two comments from stlaz and xingxingxia